### ISSUE TITLE
Can't access whole window

### ISSUE THREAD

**Issue by koumal8** [lushen124/Universal-FE-Randomizer#8]
Hello, I'm on Mac OS X High Sierra and I can't access the whole Yune window (this was true for previous versions as well): the bottom part of the window is offscreen and I have no way to bring it on screen or scroll down or resize the winndow. This has happened to me with another Java app before, what am I doing wrong? This is what I see:

![schermata 2019-01-23 alle 14 54 07](https://user-images.githubusercontent.com/40038845/51611164-ddab9400-1f1e-11e9-9999-02c1f6020473.png)

**Comment by lushen124** [comment]
You're not doing anything wrong, it's more because I wasn't quite able to keep the window as small this time around. If you're on a Macbook Pro, there's a good chance you have a scaled display as well. You can try getting rid of the scale and see if that helps bring it back on screen temporarily, though it'll make everything smaller. If you go to System Preferences > Display, you can choose the scaled option and choose to show more space to get more on screen. 

<img width="780" alt="screen shot 2019-01-23 at 10 07 14 am" src="https://user-images.githubusercontent.com/1689159/51627359-b17c1d00-1ef6-11e9-9417-5a8d0aa66da5.png">

**Comment by koumal8** [comment]
Unfortunately I'm already on max resolution, so that's not an option for me; do you think there's any chances you might implement a scroll-down feature alongside a generally smaller window in the future?

**Comment by lushen124** [comment]
Yeah, I'd have to think about how to shrink things, but I can look into it. I don't know about scrolling, but if that's what it comes down to, then I'll add it.

Just to make sure, does the GBA FE settings fit?

**Comment by lushen124** [comment]
Just to give you an update. Version 0.7.3 now accounts for this and will scroll in the case that the screen is not large enough to accommodate content. On a mac, you _may_ need to hide your dock to get to the bottom, but the other alternative is to put it into full screen mode. Regardless, it does scroll now, so give it a shot.

**Comment by koumal8** [comment]
Thanks a lot, I can scroll now and access the whole window, but btw I still can't resize it since the bottom is just out of screen (hiding the dock isn't enough); I'm on a screen with 1440x900 max resolution.

**Comment by lushen124** [comment]
Ok, I'll make the next version resizable vertically. In the mean time, you can make it full screen to access the entire window.

**Comment by lushen124** [comment]
0.7.5 is out, and it should be able to resize vertically now. I tested on a 640 high resolution, so you should be able to reach it easily.

**Comment by lushen124** [comment]
I haven't heard any other issues about this, so I'm going to assume this is fixed. If this is still an issue, please file a new ticket.
